controllers/users.js

```diff
@@ -607,6 +607,26 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+const removeTokens = async (req, res) => {
+  try {
+    const users = await userQuery.fetchUsersWithToken();
+    const length = users.length;
+
+    if (length === 0) {
+      return res.status(404).json({ message: "No users found with github Token!" });
+    }
+
+    await userQuery.removeGitHubToken(users);
+
+    return res.status(200).json({
+      message: "Github Token removed from all users!",
+      usersFoundWithToken: length,
```

Comment:
I got a feedback today that we shouldn't use too long names hence if you want you can use `usersAffected` or `usersModified` or `usersUpdated` in place of `usersFoundWithToken`

Reply:
can also directly use users.length instead of new variable length